src/psyclone/transformations.py

```diff
@@ -41,29 +41,32 @@
     checks before calling the base class for the actual transformation. '''
 
 from __future__ import absolute_import, print_function
+
 import abc
 import six
+
 from fparser.two.utils import walk
 from fparser.common.readfortran import FortranStringReader
 from fparser.two.Fortran2003 import Subroutine_Subprogram, \
     Subroutine_Stmt, Specification_Part, Type_Declaration_Stmt, \
     Implicit_Part, Comment
+
 from psyclone import psyGen
+from psyclone.configuration import Config
```

Comment:
How come these statements have all been re-ordered?

Reply:
Because our coding style clearly states that the modules in the import section need to be alphabetically sorted.
It does.
I am sure of it.
Very sure.
Nearly certain, let me check.
... silence ...
... frantically reading docs ...

Dang, it doesn't - and I was sure of it. I've been reordering imports quite a bit whenever I notice them. Reason for sorting them is to find duplicated imports, so I _think_ it should go into our coding standard.

Reply:
I'd be happy with that addition. (I've been doing it in the various `__init__` files as it really helps when we've got a lot of imports.) Unless @TeranIvy, @rupertford or @sergisiso object then please could you update the coding standard in the Dev Guide.

Reply:
Since everyone agreed - well, no one disagreed, which is the same, right - I've added a section to the coding style. I noticed that we seem to be somewhat inconsistent with the usage of backticks :( I originally had pylint and pytest in backticks, but it seems to be mostly used without, so I removed that (feel free to revert 62a10305541088af714e0e9b3383e27073107467 if you prefer having backticks :) ).